documentation/modules/oauth/proc-oauth-authorization-keycloak-example.adoc

```diff
@@ -28,7 +28,7 @@ If you want to try the example with Keycloak, use these files to perform the tas
 
 . xref:proc-oauth-authorization-keycloak-example-setup_{context}[Accessing the Keycloak Admin Console]
 . xref:proc-oauth-authorization-keycloak-example-deploy-kafka_{context}[Deploying a Kafka cluster with Keycloak authorization]
-. xref:proc-oauth-authorization-keycloak-example-authentication_{context}[Preparing TLS connectivity for a CLI Kafka client session]
+. xref:proc-oauth-authorization-keycloak-example-authentication_{context}[Preparing mTLS connectivity for a CLI Kafka client session]
```

Comment:
@mstruk - We're adopting _mTLS_ instead of _TLS client authentication_ in the docs. In https://strimzi.io/docs/operators/in-development/configuring.html#proc-oauth-authorization-keycloak-example-authentication_str are we just using TLS for encryption?

Reply:
Yes, this is TLS, not mTLS. When we are configuring truststores for 'https' connectivity to authorization server it's always TLS, not mTLS that we're talking about.